### ISSUE TITLE
Including own stylesheet in plug-in not working

### ISSUE THREAD

**Issue by MalcolmK** [mauvm/documark#7]
Hey,

when I try to load a stylesheet in a plug-in, the stylesheet is not loaded. How can this be fixed?

Malcolm

**Comment by mauvm** [comment]
Can you please describe your approach? Code fragments would help.

**Comment by MalcolmK** [comment]
Sure.

I'm now using the following snippet to include the stylesheet in my plugin:

``` node
var stylesheets = (document.config().stylesheets = (document.config().stylesheets || []));

// Add plug-in stylesheet.
stylesheets.push(path.join(__dirname, 'assets', 'style.css'));
```

When I'm testing my plugin and therefor loading the plugin from a path from my system, the stylesheet is included and works fine. But when I use the published version, it doesn't seem to work.

If you need to know anything else, ask me :)

**Comment by mauvm** [comment]
What do you mean by 'published version'? An Documark plugin (NPM package) that tries to include it's stylesheet?

If you are using the [`documark-page-meta` plugin](https://www.npmjs.com/package/documark-page-meta) you can also add a stylesheet link in the document itself:

``` js
$.root().append('<link rel="stylesheet" type="text/css" href="file://' + path.join(__dirname, 'assets', 'style.css') + '"/>');
```

Also see: https://github.com/jeroenkruis/dmp-sass/blob/master/index.js#L18

**Comment by MalcolmK** [comment]
By 'published version' I mean the version of the plugin that I have published to npm.

Okay I will try it that way.

**Comment by MalcolmK** [comment]
No, this doesn't work either.

**Comment by jeroenkruis** [comment]
Are you sure you are loading the plugin before the dmp-meta plugin executes?

**Comment by MalcolmK** [comment]
@jeroenkruis Yes, I am:

![image](https://cloud.githubusercontent.com/assets/2427717/6571382/1bf6c554-c708-11e4-9f38-8aea933d557a.png)

**Comment by mauvm** [comment]
Try this:
1. Append the link to your root (see my previous comment).
2. Log the stylesheet array in the page meta plugin file:
   - https://github.com/mauvm/dmp-page-meta/blob/836d1474615725d968c532a3c70ed5d841bdeb04/index.js#L55
   - Your local path will be something like: `node_modules/documark-theme-default/node_modules/documark-page-meta/index.js`
3. Check if your stylesheet path is included in the array.

**Comment by MalcolmK** [comment]
@mauvm With thanks to @jeroenkruis I was able to solve this problem. I have used the approach you mentioned above, by including the stylesheet like:

`$.root().append('<link rel="stylesheet" type="text/css" href="file://' + path.join(__dirname, 'assets', 'base.css') + '"/>');`

But, instead of calling stylesheet file `style.css`, I called it `base.css`. This is because when you call it `style.css` it will be overwritten.
